public/controllers/agent/components/register-agent.js

```diff
@@ -616,72 +914,402 @@ export const RegisterAgent = withErrorBoundary(
+                    {this.systemSelectorWazuhControlMacos()}
+                  </EuiCodeBlock>
+                  <EuiCopy textToCopy={this.systemSelectorWazuhControlMacos()}>
+                    {(copy) => (
+                      <div className="copy-overlay" onClick={copy}>
+                        <p><EuiIcon type="copy" /> Copy command</p>
+                      </div>
+                    )}
+                  </EuiCopy>
+                </div>
+                <EuiSpacer size='s' />
+                {textAndLinkToCheckConnectionDocumentation}
+              </EuiText>
+            </Fragment>
+          ),
+        },
+      ];
+
+      const tabWazuhControl = [
+        {
+          id: 'Wazuh-control',
+          name: 'Wazuh-control',
+          content: (
+            <Fragment>
+              <EuiSpacer />
+              <EuiText>
+                <div className="copy-codeblock-wrapper">
+                  <EuiCodeBlock style={codeBlock} language={language}>
+                    {this.systemSelectorWazuhControl()}
+                  </EuiCodeBlock>
+                  <EuiCopy textToCopy={this.systemSelectorWazuhControl()}>
+                    {(copy) => (
+                      <div className="copy-overlay" onClick={copy}>
+                        <p><EuiIcon type="copy" /> Copy command</p>
+                      </div>
+                    )}
+                  </EuiCopy>
+                </div>
+                <EuiSpacer size='s' />
+                {textAndLinkToCheckConnectionDocumentation}
+              </EuiText>
+            </Fragment>
+          ),
+        },
+      ];
+
+      const buttonGroup = (legend, options, idSelected, onChange) => {
+        return (
+          <EuiButtonGroup
+            color="primary"
+            legend={legend}
+            options={options}
+            idSelected={idSelected}
+            onChange={onChange}
+            className={'osButtonsStyle'} />
+        )
+      }
+
+      const buttonGroupWithMessage = (legend, options, idSelected, onChange) => {
+        return (
+          <>
             <EuiButtonGroup
               color="primary"
-              legend="Choose the Operating system"
-              options={osButtons}
-              idSelected={this.state.selectedOS}
-              onChange={(os) => this.selectOS(os)}
+              legend={legend}
+              options={options}
+              idSelected={idSelected}
+              onChange={onChange}
+              className={'osButtonsStyle'}
             />
+            {this.state.selectedVersion == 'solaris10' || this.state.selectedVersion == 'solaris11' ? <EuiCallOut color="warning" className='message' iconType="iInCircle" title={
+              <span>
+                Might require some extra installation<EuiLink target="_blank" href="https://documentation.wazuh.com/current/installation-guide/wazuh-agent/wazuh-agent-package-solaris.html"> steps</EuiLink>.
```

Comment:
**nitpick**: The links to the documentation should use `webDocumentationLink` whose prefix includes the Wazuh version (major.minor) instead of `current`.

Reply:
Fixed